be/src/storage/lake/tablet_manager.h

```diff
@@ -21,9 +23,11 @@ template <typename T>
 class MetadataIterator;
 using TabletMetadataIter = MetadataIterator<TabletMetadataPtr>;
 using TxnLogIter = MetadataIterator<TxnLogPtr>;
+using TabletSchemaPtr = std::shared_ptr<const starrocks::TabletSchema>;
 
 class TabletManager {
     friend class Tablet;
+    friend class starrocks::ExecEnv;
```

Comment:
It's a little bit strange to make ExecEnv as a friend class of `TabletManager`, you can define `prune_metacache()` as a public method and remove the friend class declaration.